pandas/tests/series/methods/test_replace.py

```diff
@@ -373,3 +373,19 @@ def test_replace_invalid_to_replace(self):
         )
         with pytest.raises(TypeError, match=msg):
             series.replace(lambda x: x.strip())
+
+    def test_replace_only_one_dictlike_arg(self):
+        # GH#33340
```

Comment:
just increasing test coverage? or is this an actual bug?

Reply:
the new helper method is DataFrame-only, so this PR adds a if-series-then-raise check.  this test hits that